pkg/api/pod/util.go

```diff
@@ -842,3 +844,14 @@ func multiplePodIPsInUse(podStatus *api.PodStatus) bool {
 	}
 	return false
 }
+
+// GetPodPriority returns priority of the given pod.
+func GetPodPriority(pod *v1.Pod) int32 {
```

Comment:
Why are we moving this function here?

Reply:
The podinfo package has exactly the same function.

Reply:
Should I keep it here or in podinfo?

Reply:
> Should I keep it here or in podinfo?

We should keep it in podinfo if it works well.

Reply:
+1

Reply:
wait, this is what the kubelet uses. But, this should actually go to `pkg/api/v1/pod`.

You could consider open a separate PR just for moving this function, as it involves a different set of reviewers.

Reply:
Closing this PR and opening separate PR